collectcontexts.py

```diff
@@ -75,10 +105,11 @@ def main():
             help='Location of the BHSA data')
     p_data.add_argument('--module', '-m', nargs=1, required=True,
             help='Text-fabric module to load')
+    p_data.add_argument('--lang', '-l', nargs=1, required=True,
```

Comment:
If you leave out `nargs=1`, the parser should simply produce the string, rather than a list of one element (see https://docs.python.org/3/library/argparse.html#nargs). Then you can simply use `lang` instead of `lang[0]` in `gather`.